doc/swagger.yaml

```diff
@@ -86,7 +86,7 @@ paths:
           description: Invalid input
       security:
         - APIKeyHeader: []
-  '/dependencies/{dependencyId}/comments/{commentId}':
+  '/comments/{comments_id}':
```

Comment:
I think it should be `{comment_id}`, singular. Or even better: just `id` :trophy: